providers/base/units/miscellanea/jobs.pxu

```diff
@@ -489,10 +489,19 @@ category_id: com.canonical.plainbox::miscellanea
 estimated_duration: 60
 id: miscellanea/ubuntu-desktop-recommends
 requires: package.name == 'ubuntu-desktop'
-command: check-ubuntu-desktop-recommends.sh
+command: check-ubuntu-desktop-recommends.sh ubuntu-desktop
 _summary: Check if Debian packages in Recommends of ubuntu-desktop are installed
 _description: Check if Debian packages in Recommends of ubuntu-desktop are installed
 
+plugin: shell
+category_id: com.canonical.plainbox::miscellanea
+estimated_duration: 60
+id: miscellanea/ubuntu-desktop-minimal-recommends
+requires: package.name == 'ubuntu-desktop-minimal'
+command: check-ubuntu-desktop-recommends.sh ubuntu-desktop-minimal
+_summary: Check if Debian packages in Recommends of ubuntu-desktop-minimal are installed
```

Comment:
This summary could be improved. What about `Check that all the recommended packages for ubuntu-desktop-minimal are installed`?

(Maybe it would be good to modify the summary of the `miscellanea/ubuntu-desktop-recommends` as well to match)